api/src/utils/pipes/search-filter.pipe.ts

```
@@ -143,6 +158,35 @@ export class SearchFilterPipe<T>
           };
+        case 'in':
+          // Handle $in operator - convert to MongoDB $in syntax
+          const inQuery = Object.entries(data || {}).reduce(
+            (inAcc, [field, values]) => {
+              return {
+                ...inAcc,
+                [field]: { $in: values },
+              };
+            },
+            {},
+          );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)